app/controllers/notes_controller.rb

```diff
@@ -32,6 +32,11 @@ def create
   # Retrieve a Note given its :id
   def show
     @activities = @note.activities.latest
+    @subscription = Subscription.find_by(
```

Comment:
@dormi We'll be seeing a lot of this `find_by` method called the same way. Do you think it's worth it to add an instance method to the `Subscribable` module that does something like this?
e.g.
```
def current_subscription(user)
  Subscription.find_by(
    user: user,
    subscribable_type: self.class.to_s,
    subscribable_id: self.id
  )
end
```